administrator/components/com_finder/src/Helper/LanguageHelper.php

```diff
@@ -58,8 +58,9 @@ public static function branchSingular($branchName)
     {
         $return   = preg_replace('/[^a-zA-Z0-9]+/', '_', strtoupper($branchName));
         $language = Factory::getApplication()->getLanguage();
+        $debug = Factory::getApplication()->get('debug_lang');
```

Comment:
```suggestion
        $debug    = Factory::getApplication()->get('debug_lang');
```
Fix php-cs-fixer failing in Drone.